packages/dmn-editor/src/propertiesPanel/FontOptions.tsx

```diff
@@ -374,22 +373,24 @@ export function FontOptions({ startExpanded, nodeIds }: { startExpanded: boolean
               </ToggleGroup>
             </div>
             <br />
-            <Select
-              toggleRef={toggleRef}
-              variant={SelectVariant.single}
-              aria-label={"Select font style"}
-              isOpen={isFontFamilySelectOpen}
-              onSelect={onSelectFont}
-              onToggle={() => setFontFamilySelectOpen((prev) => !prev)}
-              selections={fontFamily ?? ""}
-              isDisabled={false}
-              maxHeight={inViewTimezoneSelect.maxHeight}
-              direction={inViewTimezoneSelect.direction}
-            >
-              {WEBSAFE_FONTS_LIST.map((fontName, index) => (
-                <SelectOption key={index} value={fontName} style={{ fontFamily: fontName }} />
-              ))}
-            </Select>
+            <div data-testid="node-font-style-selector">
+              <Select
+                toggleRef={toggleRef}
+                variant={SelectVariant.single}
+                aria-label={"Select font style"}
+                isOpen={isFontFamilySelectOpen}
+                onSelect={onSelectFont}
+                onToggle={() => setFontFamilySelectOpen((prev) => !prev)}
+                selections={fontFamily ?? ""}
+                isDisabled={false}
+                maxHeight={inViewTimezoneSelect.maxHeight}
+                direction={inViewTimezoneSelect.direction}
+              >
+                {WEBSAFE_FONTS_LIST.map((fontName, index) => (
+                  <SelectOption key={index} value={fontName} style={{ fontFamily: fontName }} />
+                ))}
+              </Select>
+            </div>
```

Comment:
We can not simply add `data-testid` int `Select`, patternfly for some reason ignores that attribute.

Reply:
Probably we can not, `<React.Fragment data-testid={"xyz"}>` has no effect for the produced html.  The `data-testid` is not in the generated html at all.